### DISCUSSION TITLE
Issue with reset password token

### DISCUSSION THREAD

**Discussion by benjifs** [bootstrap-email/bootstrap-email#145] [Help] [answered]
Minor issue but I tried the password reset feature and received a link to:

http://editor.bootstrapemail.com/users/password/edit?reset_password_token=██████

which redirects to the editor.

The subdomain in the reset password link should be changed to `app` instead to be:

http://app.bootstrapemail.com/users/password/edit?reset_password_token=██████

I couldn't find a repo for the editor to submit the issue directly so hopefully this is ok

**Comment by stuyam** [comment, marked as answer]
Fixed 👍 thanks for bringing this to my attention!
